src/components/Storefront/Footer.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from 'react';
+import { Image, Center, Text, Link, Button, HStack, VStack, Input } from '@chakra-ui/react';
+import { ChevronRightIcon } from '@chakra-ui/icons';
+import { FaFacebookSquare, FaInstagramSquare, FaYoutube, FaLinkedin, FaTwitter } from 'react-icons/fa';
+import smallLogo from '../../assets/building_up_footer_small_icon.png';
+
+const Footer = props => {
+  const { subscribtionEmail, changeSubscriptionEmail } = useState();
+
+  const subscribe = e => {
+    e.preventDefault();
+    changeSubscriptionEmail(e);
+  };
+
+  const updateSubscriptionEmail = e => {
+    e.preventDefault();
+    alert(subscribtionEmail);
+  };
+
+  return (
+    <VStack spacing="0px">
+      <HStack backgroundColor="black" w="100%" display="flex" paddingTop="54px" paddingBottom="84px">
+        <Image src={smallLogo} alt="Small Icon" marginLeft="41px" marginRight="47px" alignSelf="flex-start" />
+        <HStack backgroundColor="black" w="100%" display="flex" justifyContent="space-between">
+          <VStack w="13%" alignItems="flex-start" spacing="26px">
```

Comment:
would prefer not to manually set the width here, just leave the width to be set automatically by the width of the children